lib/constants.rb

```diff
@@ -116,8 +116,7 @@ module Constants
   ].freeze
 
   CONTENT_TYPES = {
-    'book (ltr)' => Cocina::Models::ObjectType.book,
-    'book (rtl)' => Cocina::Models::ObjectType.book,
+    'book' => Cocina::Models::ObjectType.book,
     'file' => Cocina::Models::ObjectType.object,
```

Comment:
we are now using the constant to create the `Process : Content Type` on the server side (it used to be generated by the javascript before, but now our html has the cocina types and not strings)